src/core/annotations/one-of.ts

```diff
@@ -27,8 +27,8 @@ function OneOfFactory(values: any[]) {
         }
       }
     });
-  }
+  };
 }
 
 
-export const OneOf = OneOfFactory;
+export const oneOf = oneOfFactory;
```

Comment:
@jelbourn Already discussing other options with Hans. Nobody is a fan of it, to include me.